src/configHandler.test.js

```diff
@@ -0,0 +1,176 @@
+const test = require('ava');
+const sinon = require('sinon');
+const fs = require('fs');
+
+const configHandler = require('./configHandler');
+
+const testConfig = `
+tvheadend_url: http://admin:test@192.168.0.1:9981
+antennas_url: http://127.0.0.1:5004
+tuner_count: 6 # numbers of tuners in tvheadend
+device_uuid: 2f70c0d7-90a3-4429-8275-cbeeee9cd605
+`.trim();
+
+let sandbox;
+let fsStub;
+
+test.beforeEach(() => {
+  sandbox = sinon.createSandbox();
+  fsStub = sandbox.stub(fs);
+});
+
+test.afterEach.always(() => {
+  sandbox.restore();
+});
+
+test.serial('loads the specified config', t => {
+  fsStub.existsSync.returns(true);
+  fs.readFileSync.returns(testConfig);
+  const results = configHandler.loadConfig('test.yml');
+
+  t.is(results.tvheadend_parsed_uri, 'http://192.168.0.1:9981');
+  t.is(results.tvheadend_username, 'admin');
+  t.is(results.tvheadend_password, 'test');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
  t.is(results.tvheadend_password, "test");
```